content/en/docs/Reference/file-based-catalogs.md

```diff
@@ -304,6 +304,49 @@ properties:
     kind: Bar
 ```
 
+#### `olm.deprecations`
+
+Operator authors can use the file-based catalog (FBC) deprecation schema to provide relevant information for support and upgrades for users running operators. The deprecation schema follows the Meta FBC Schema conventions. 
+
+You can use the `olm.deprecations` schema to deprecate packages, bundles, and channels. This schema consists of a list of scoped references to catalog content, with a descriptive message string that provides more information to the user. The deprecation schema is an optional feature.
+
+A valid deprecation schema meets the following criteria:
+- There must be only one schema per package
+- The message must be a non-zero length
+- The package name must exist in the bundle
+  
+The deprecation feature does not consider overlapping deprecation (package vs channel vs bundle).
+
+|              | `olm.package`                   | `olm.channel`        | `olm.bundle`          |
+|--------------|---------------------------------|----------------------|-----------------------|
+| Scope        | Entire Package                  | Single Channel       | Single Bundle Version |
+| Requirements | `name` must be empty | `name`  is mandatory | `name`  is mandatory  |
```

Comment:
```suggestion
| Requirements | `name` must be empty since it is inferred from the parent `package` field | `name`  is mandatory | `name`  is mandatory  |
```